code/game/machinery/computer/computer.dm

```diff
@@ -33,8 +33,8 @@
 
 /obj/machinery/computer/process()
 	if(stat & (NOPOWER|BROKEN))
-		return 0
-	return 1
+		return FALSE
+	return TRUE
```

Comment:
I realise this is entirely unrelated to the PR, but I changed this while testing if `process()` worked rather than `ui_data()` and forgot to remove it so... why not just keep it at this point.